03-use-cases/01-finance/retail-banking/main.ipynb

```diff
@@ -0,0 +1,1931 @@
+{
```

Comment:
<p>🧜🏽‍♀️ got us again :(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/824/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Will fix this in the upcoming upgrade for `0.8.10`